packages/sdk/client/src/invitations/invitations-proxy.ts

```diff
@@ -156,6 +156,24 @@ export class InvitationsProxy implements Invitations {
     return observable;
   }
 
+  async resumePersistentInvitations() {
```

Comment:
What happens here if two separate clients pointing to the same service call this? Are the invitaitons gonna be doubled?

I think should should be a part of the service-host startup sequence instead.

Reply:
hmm, good question. I don't think the invitation will be duplicated because createInvitation checks for existing invitations.

not sure about what the implications of multiple observables are, it might actually be beneficial so both clients get notified when an invitation reaches a terminal state. how would that work if this moves to the service-host startup?